interface/orders/orders_results.php

```diff
@@ -616,6 +622,18 @@ class='form-control'
                                 class='form-control'
                                 value='<?php echo attr($result_text); ?>' />
                         </td>
+                        <td class="text-nowrap">
+                                <input type='text' size='13' name='form_result_date[<?php echo attr($lino); ?>]'
+                                    id='form_result_date[<?php echo attr($lino); ?>]'
+                                    class='form-control datetimepicker' value='<?php echo attr($date_result); ?>'
+                                    title='<?php echo xla('Date and time of this result'); ?>' />
+                        </td>
+                        <td class="text-nowrap">
+                                <input type='text' size='13' name='form_result_date_end[<?php echo attr($lino); ?>]'
+                                    id='form_result_date_end[<?php echo attr($lino); ?>]'
+                                    class='form-control datetimepicker' value='<?php echo attr($date_result_end); ?>'
```

Comment:
This needs to be $result_date_end instead of $date_result_end.  That makes it so the date shows up on the save.